commercetools/commercetools-sdk-java-api/src/integrationTest/java/commercetools/product/ProductIntegrationTests.java

```diff
@@ -117,6 +117,23 @@ public void query() {
         });
     }
 
+    @Test
```

Comment:
probably it's better to add ignore because for this test the Search has to be enabled